qt/python/mantidqt/widgets/workspacedisplay/matrix/presenter.py

```diff
@@ -189,20 +189,34 @@ def _do_action_plot(self, table, axis, get_index, plot_errors=False):
 
         ws_list = [self.model._ws]
         self.plot(ws_list, wksp_indices=[get_index(index) for index in selected], errors=plot_errors,
-                  plot_kwargs=plot_kwargs)
+                  overplot=overplot, plot_kwargs=plot_kwargs)
 
     def action_plot_spectrum(self, table):
         self._do_action_plot(table, MantidAxType.SPECTRUM, lambda index: index.row())
 
     def action_plot_spectrum_with_errors(self, table):
         self._do_action_plot(table, MantidAxType.SPECTRUM, lambda index: index.row(), plot_errors=True)
 
+    def action_overplot_spectrum(self, table):
+        self._do_overaction_plot(table, MantidAxType.SPECTRUM, lambda index: index.row(),
```

Comment:
I'm a bit confused with this line as _do_overaction_plot doesn't exist yet from what I've tested it works fine. Does this mean this is not called at all?

Reply:
Yeah I'm also confused. This a typo 🤦‍♂️ . But also wondering why its working.....I'll have a look in the afternoon